docs/devenv_rhel8.md

```diff
@@ -142,13 +142,14 @@ It should now be possible to run a standalone MicroShift executable file as pres
 ### Running MicroShift
 Run the MicroShift executable file in the background using the following command.
 ```bash
-nohup sudo ~/microshift/microshift run >> /tmp/microshift.log &
+nohup sudo ~/microshift/_output/bin/microshift run >> /tmp/microshift.log &
 ```
 Examine the `/tmp/microshift.log` log file to ensure a successful startup.
 
 > An alternative way of running MicroShift is to update `/usr/bin/microshift` file and restart the service. The logs would then be accessible by running the `journalctl -xu microshift` command.
 > ```bash
-> sudo cp -f ~/microshift/microshift /usr/bin/microshift
+> sudo systemctl stop microshift
```

Comment:
Nit: Instead of a "stop" here, how about a "restart" below? Would then also match the instructions in the text.